src/sbd-cluster.c

```diff
@@ -203,6 +281,7 @@ static gboolean
 sbd_get_two_node(void)
```

Comment:
probably should rename

Reply:
yep - especially when going for independent switching so that it might not even do anything with two_node.
guess verify_against_cmap_config() is a good description for what it is doing for now.